src/components/ContentComponent/_tests_/ContentComponent.test.js

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import ContentComponent from "../index";
+
+describe('Content Component tests', () => {
+    it('Should render Content Component with subtitle & content', () => {
+        render(<ContentComponent />);
```

Comment:
If a component has props better to use them to test. The test below will check for that the `p` tag exist. But testing that the content/subtitle shows up on the page is more valuable.

Since you have a conditional with the button, you can have two tests for it. Add `noBtn` prop to test no button appears and vice versa.

```
render(<ContentComponent subtitle="subtitle" content="content" />);
expect(screen.getByText('subtitle').toBeInTheDocument()
expect(screen.getByText('content').toBeInTheDocument()
expect(screen.getByText('Learn More').toBeInTheDocument()
```